compiler_opt/rl/corpus.py

```diff
@@ -34,77 +35,93 @@ def build_modulespecs_from_datapath(
     additional_flags: Tuple[str, ...] = (),
     delete_flags: Tuple[str, ...] = ()
 ) -> List[ModuleSpec]:
-  module_paths: List[str] = _load_module_paths(data_path)
-
-  has_thinlto: bool = _has_thinlto_index(module_paths)
+  # TODO: (b/233935329) Per-corpus *fdo profile paths can be read into
+  # {additional|delete}_flags here
+  metadata: Dict[str, any] = _load_metadata(
+      os.path.join(data_path, 'metadata.json'))
```

Comment:
instead of `metadata` for the name, let's use `corpus_description`: it's more specific.